### ISSUE TITLE
Throttling / Rate Limiting

### ISSUE THREAD

**Issue by 3zzy** [Exeu/apai-io#67]
I keep getting:

    <b>Fatal error</b>:  Uncaught exception 'RuntimeException' with message 'An error occurred while     sending request. Error number: 28; Error message: Operation timed out after 10000 milliseconds with     5583 bytes received' in /vendor/exeu/apai-io/lib/ApaiIO/Request/Rest/Request.php:172
    Stack trace:
    #0 /vendor/exeu/apai-io/lib/ApaiIO/ApaiIO.php(73): ApaiIO\Request\Rest\Request-&gt;perform(Object(ApaiIO\Operations\Search))
    #1 amazon.php(81): ApaiIO\ApaiIO-&gt;runOperation(Object(ApaiIO\Operations\Search))
    #2 amazon.php(58): searchItem('INTERMEC', 'Hand Strap for ...', 35.442, 'com')
    #3 combined.php(103): getItem('INTERMEC', 'Hand Strap for ...', 35.442, 'com')
    #4 {main}
      thrown in <b>/vendor/exeu/apai-io/lib/ApaiIO/Request/Rest/Request.php</b> on line <b>172</b><br />

.. is there any way to rate limit the requests to `1 request / sec` which is the current limit.

**Comment by Exeu** [comment]
No this is currently not supported by this lib.
And this will not be included in the near future.

Before starting a new ticket please have a look on #65 because this topic is not new.
